src/vmx.rs

```diff
@@ -756,6 +756,84 @@ pub fn read_flags() -> u64 {
     ret
 }
 
+#[naked]
```

Comment:
Not really a review comment, but it's not clear to me where this function gets called and why it needs to be naked (as opposed to extern "C"). Can you explain it to me?